app/controllers/api/v1/booking_requests_controller.rb

```diff
@@ -6,14 +6,22 @@ class BookingRequestsController < ActionController::Base
       before_action :authorise_api_user!
 
       def create
-        booking_request = BookingRequest.create!(booking_request_params)
-        BookingRequests.customer(booking_request).deliver_later
+        booking_request = BookingRequest.new(booking_request_params)
 
-        head :created
+        if booking_request.save
+          BookingRequests.customer(booking_request).deliver_later
+          head :created
+        else
+          render_errors(booking_request)
+        end
       end
 
       private
 
+      def render_errors(booking_request)
+        render json: { errors: booking_request.errors }, status: :unprocessable_entity
```

Comment:
>Does this correctly format slot validations or do we just get slot is invalid?

>Does this group the slots priority validation from the booking request object with the validations from the slot object?

Yes. The output looks something like this:

```javascript
{
  "errors": {
    "name": [
      "can't be blank"
    ],
    "email": [
      "can't be blank"
    ],
    "phone": [
      "can't be blank"
    ],
    "memorable_word": [
      "can't be blank"
    ],
    "age_range": [
      "is not included in the list"
    ],
    "accessibility_requirements": [
      "is not included in the list"
    ],
    "marketing_opt_in": [
      "is not included in the list"
    ],
    "defined_contribution_pot": [
      "is not included in the list"
    ],
    "slots": [
      "you must provide a slot for each permitted priority (1, 2, 3)"
    ]
  }
}
```

>Do we care (i.e. are we planning to use these validation messages anywhere)?

At this stage, no. We'll log them to diagnose any issues that may occur, but the likelihood of requests getting this far is entirely exceptional circumstance, and would highlight an issue in the client. There will be a time when locations are supplanted by something over here - likely when booking centres need to manage their own slots and availability - at which time this will feel somewhat more cohesive.